src/Sarif.Multitool/Rules/MessagesShouldEndWithPeriod.cs

```diff
@@ -9,7 +9,12 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
 {
     public class MessagesShouldEndWithPeriod : SarifValidationSkimmerBase
```

Comment:
>MessagesShouldEndWithPeriod [](start = 17, length = 27)

Treat this as new code. The replacement of simple message strings with `message` object caused a lot of churn in this rule.